lms/static/styles/components/_FileList.scss

```diff
@@ -61,10 +54,14 @@ $file-list-icon-size: 24px;
   padding-right: 10px;
 }
 
-.FileList__no-files {
-  border: 1px solid var.$grey-6;
-  height: 40px;
+.FileList__no-files-message {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
   vertical-align: middle;
-  padding: 12px;
+  text-align: center;
+  width: 100%;
+  min-height: 149px;
```

Comment:
Where does the `min-height` come from here? Is there a way of calculating it from other data? Could it vary depending on the browser or system-specific settings like the font used?